src/operator/contrib/dgl_graph.cc

```diff
@@ -35,6 +35,833 @@
 namespace mxnet {
 namespace op {
 
+typedef int64_t dgl_id_t;
+
+////////////////////////////// Graph Sampling ///////////////////////////////
+
+unsigned int seed = 123;
```

Comment:
we should't fix a seed like this.

Reply:
I think we need to use mxnet-random here.